src/zenml/integrations/integration.py

```diff
@@ -50,13 +49,17 @@ class Integration(metaclass=IntegrationMeta):
     def check_installation(cls) -> bool:
         """Method to check whether the required packages are installed"""
         try:
-            for req, command in cls.SYSTEM_REQUIREMENTS.items():
+            for requirement, command in cls.SYSTEM_REQUIREMENTS.items():
```

Comment:
I think @schustmi already has a branch for this :-)

Reply:
I merged @schustmi 's branch before the request so it should be good to go.